packages/app/src/runner/SpecRunner.vue

```diff
@@ -158,7 +178,7 @@ $navbar-width: 80px;
 
 <style>
 #unified-runner > div {
-  height: 100%;
+  height: min(100%, 100vh);
```

Comment:
Context on this from last time we implemented `cy.screenshot` in `runner-ct` (which is slightly different to E2E since the DOM hierarchy looks so different): https://github.com/cypress-io/cypress/pull/16543/files#r634296671